homeassistant/components/isy994/binary_sensor.py

```diff
@@ -81,6 +85,15 @@ def _detect_device_type(node) -> str:
     split_type = device_type.split('.')
     for device_class, ids in ISY_DEVICE_TYPES.items():
         if '{}.{}'.format(split_type[0], split_type[1]) in ids:
+            # Insteon Thermostats have 2 sub-notes, one for Cool Control and
+            # one for Heat Control.
+            if device_class == 'climate':
+                subnode_id = int(node.nid[-1], 16)
+                # pylint: disable=no-else-return
+                if subnode_id == 3:
+                    return 'heat'
+                else:
```

Comment:
Remove the `else:` and we can outdent the `return 'cold'` below. Then we can remove the pylint disable.

Reply:
Corrected.  Sorry, I thought that had been removed. May have lost the change during the rebase.